extract_assets.py

```diff
@@ -21,7 +22,13 @@ def ExtractFile(xmlPath, outputPath, outputSourcePath):
         # Don't extract if another file wasn't extracted properly.
         return
 
-    execStr = f"tools/ZAPD/ZAPD.out e -eh -i {xmlPath} -b baserom/ -o {outputPath} -osf {outputSourcePath} -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml {ZAPDArgs}"
+    zapdPath = Path("tools") / "ZAPD" / "ZAPD.out"
```

Comment:
This fixes:
![image](https://user-images.githubusercontent.com/5306619/235380620-e601643a-08d6-4cf7-b747-e74f7596eada.png)